src/NuGet.Clients/NuGet.SolutionRestoreManager.Interop/NuGet.SolutionRestoreManager.Interop.csproj

```diff
@@ -3,23 +3,28 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), 'README.md'))\build\common.props" />
 
   <PropertyGroup>
+    <PackageId>NuGet.SolutionRestoreManager.Interop</PackageId>
+    <PackageVersion>$(Version)</PackageVersion>
     <TargetFramework>net46</TargetFramework>
     <Shipping>true</Shipping>
     <PackProject>true</PackProject>
     <IncludeInVsix>true</IncludeInVsix>
+    <authors>Microsoft</authors>
```

Comment:
if at other places the author is NuGet , we should change this to NuGet as well

Reply:
It's Microsoft for the interops only. 
That makes me hesistant to change it.